### PR TITLE
ft. : explore , bookmarks posts with some changes

### PR DESCRIPTION
**Pull request by manojsarna**
- [x] Explore Page Added
- [x] Bookmarks Page Added with functionality

Live Link : https://eloquent-croquembouche-527609.netlify.app/

###Check these Files 

[Bookmarks.jsx](https://github.com/manojsarna/manoj-smasher-socialmedia-react/compare/explore-bookmarks-functionality?expand=1#diff-718ae2625eee517956769d715d18ab79815824f51c585009a9e38fc52dea9529)

[Explore.jsx](https://github.com/manojsarna/manoj-smasher-socialmedia-react/compare/explore-bookmarks-functionality?expand=1#diff-eb14639494262521fa3ad99264df6b53017b035eadcae50a39b8e5376036137d)

### Ignore Rest of the Files

image

### CODE REVIEW

File: src/pages/explore/Explore.jsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+import { useSelector } from "react-redux";
+import { LeftSideBar, Loader, Post, RightSideBar } from "../../components";
 import "./explore.css";
 export function Explore() {
+  const posts = useSelector((state) => state.posts.posts);
+  const loading = useSelector((state) => state.posts.loading);
+  let finalPosts = [...posts];
+  finalPosts = finalPosts.sort(
+    (a, b) => new Date(b.createdAt) - new Date(a.createdAt)
+  );
```
Comment (pankaj-prog) [inline src/pages/explore/Explore.jsx:10]: suggestion: you could name it sortedPostList

File: src/components/avatar-card/avatarcard.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.sm-avatar-link-info {
+  display: flex;
+  flex-direction: column;
+}
+.sm-avatar-link-name {
+  font-size: 1.8rem;
+  font-weight: 800;
+  white-space: nowrap;
+}
```
Comment (PuneetP16) [inline src/components/avatar-card/avatarcard.css:20]: Suggestion: format the code properly, and keep equal line spacing between CSS's two rules.

File: src/pages/bookmarks/Bookmarks.jsx [hunk 3]
```diff
@@ -1,12 +1,32 @@
-import { LeftSideBar, RightSideBar } from "../../components";
 import "./bookmarks.css";
+import { useSelector } from "react-redux";
+import { LeftSideBar, Loader, Post, RightSideBar } from "../../components";
```
Comment (pankaj-prog) [inline src/pages/bookmarks/Bookmarks.jsx:3]: you could use absolute imports

**Review by pankaj-prog — APPROVED** [review]
suggested minor changes in the files mentioned.

**Review by PuneetP16 — APPROVED** [review]
The Code structure is good and also the UI. Check the mentioned comment.

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ import "./footer.css";
 export function Footer() {
   return (
     <>
-      <footer className="sm-footer">
+      {/* <footer className="sm-footer">
```
Comment (iRohitGaur) [inline src/components/footer/Footer.jsx:13]: suggestion: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! 

UI looks great.

Good to merge 👍🏻

**Pull request merged**